boards/launchxl/Makefile-app

```diff
@@ -0,0 +1,17 @@
+# Makefile for loading applications for the sensortag
+
+$(call check_defined, TOCK_USERLAND_BASE_DIR)
+$(call check_defined, BUILDDIR)
+$(call check_defined, PACKAGE_NAME)
+
+TOCK_ARCH = cortex-m4
+BOARD_BUILDDIR = $(BUILDDIR)/$(TOCK_ARCH)
+
+# Upload programs to sensortag
+.PHONY: flash
+flash: $(BOARD_BUILDDIR)/$(TOCK_ARCH).bin $(BUILDDIR)/$(PACKAGE_NAME).tab
+    $(error Cannot flash yet.)
```

Comment:
I think you should update this with the correct rule for flashing an app. Something like `@$(DSLITE) flash -c jtag/cc2652r1f.ccxml -e -f $<,$(PROG_BASE)`

Reply:
Makes sense!